be/src/runtime/load_stream_writer.cpp

```diff
@@ -154,6 +154,8 @@ Status LoadStreamWriter::close_segment(uint32_t segid) {
 
 Status LoadStreamWriter::add_segment(uint32_t segid, const SegmentStatistics& stat,
                                      TabletSchemaSPtr flush_schema) {
+    SCOPED_SWITCH_THREAD_MEM_TRACKER_LIMITER(
```

Comment:
should not use it here. It maybe abuse. It should only be used in deconstructor.

Reply:
fixed, `LoadStreamWriter::close` also used it